fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -250,32 +261,29 @@ def dense_sst_to_dst(self, dense: Tensor, sst: Tensor) -> Tensor:
             (Tensor):
                 Same shaped tensor, still dense format but has zeros. Non-zeros are top-k delta values.
         """
-        pass
+        delta = dense - self.sst_dst_to_dense(sst)  # sst_dst_to_dense(sst) returns the inverse transform here
```

Comment:
I would move this line to after 266 just before its use.
Also, I would add:

del dense

after this line so that the dense tensor could be freed it no other reference to it exists to save memory.